asn1c_combined/FixIt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+asn1c -Wdebug-compiler -fcompound-names -gen-PER -pdu=all 1609dot2-base-types.asn 1609dot2-schema.asn J2735_201603DA.ASN 2>&1 | tee compile.out
```

Comment:
Please include the file SEMI_v2.3.0_070616.asn on the asn1c command line. The files is in the sae_j2735_SEMI_api directory.